tokens/finn.no/broadcast.yml

```diff
@@ -3,7 +3,7 @@ token: maps
 
 color:
   broadcast:
-    text: gray-700
-    background: yellow-50
-    border: yellow-300
-    icon: yellow-600
+    text: s-color-text
+    background: s-color-background-warning-subtle
+    border: s-color-border-warning-subtle
+    icon: s-color-icon-warning
```

Comment:
I see that broadcast tokens were originally added to [@warp-ds/tokens](https://github.com/warp-ds/tokens/pull/26) and they are present in the spreadsheet but I don't see where they are used. Broadcast component renders a `w-toast` component of variant `warning`, which applies the toast tokens. Broadcast component itself doesn't use any colors 🤔 It doesn't hurt to keep this change but if my thinking is correct, we might want to remove `broadcast.yml` files.

Reply:
Ah. Good catch! Only thing then would be if we for any reason would like to be able to separate the style of broadcast from the toast style. 🤷 But then the broadcast component needs to implement these tokens. 😅  Let's keep them since they are already there... and we can decide when @adidick is back. Wdyt? 🙂